encoding.go

```diff
@@ -172,7 +196,25 @@ func GetEncoder(req *Request, w io.Writer, def EncodingType) (encType EncodingTy
 		fn, ok = Encoders[encType]
 	}
 	if !ok {
-		return encType, nil, Errorf(ErrClient, "invalid encoding: %s", encType)
+		// In addition to the explicitly named legacy encoders defined in this file,
+		// support encoding with the encoders registered in the ipld-prime multicodec
+		// registry.
+		ipldCodec, ok := mc.Of(string(encType))
+		if !ok {
+			if !ok {
+				n, err := strconv.Atoi(string(encType))
```

Comment:
This seems like two new features:

- more string names supported
- base-10 numbers supported

Are there docs changes that should go with it?